src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkDocstringsWillBeIgnoredInspection.kt

```diff
@@ -0,0 +1,33 @@
+package com.jetbrains.snakecharm.inspections
+
+import com.intellij.codeInspection.LocalInspectionToolSession
+import com.intellij.codeInspection.ProblemsHolder
+import com.jetbrains.snakecharm.SnakemakeBundle
+import com.jetbrains.snakecharm.lang.psi.*
+
+class SmkDocstringsWillBeIgnoredInspection : SnakemakeInspection() {
+    override fun buildVisitor(
+        holder: ProblemsHolder,
+        isOnTheFly: Boolean,
+        session: LocalInspectionToolSession
+    ) = object : SnakemakeInspectionVisitor(holder, getContext(session)) {
+
+        override fun visitSmkRule(rule: SmkRule) {
+            visitSMKRuleLike(rule)
+        }
+
+        override fun visitSmkCheckPoint(checkPoint: SmkCheckPoint) {
+            visitSMKRuleLike(checkPoint)
+        }
+
+        fun visitSMKRuleLike(rule: SmkRuleLike<SmkArgsSection>) {
+            val docstrings = rule.getStringLiteralExpressions().drop(1)
```

Comment:
Seems it is a bit complicated way (create new Array list without one element) and more strait-forward  is to check array list length.

Reply:
But after that,  I call `registerProblem` for all remaining elements (except the first one, which was dropped). Also I think it's wrong to highlight the first string expression too, because it was placed correctly. If you are agree, we'll need to drop the first element in any case. So, I think there is no other ways to simplify it.

Reply:
Ok, let's leave code as is for simplicity. There should be no noticeable performance impact here. 
P.S.: below is just a loop that iterates over docstrings. It could use indexes `1..docstrings.size -1`, coping data to new memory location is not needed for such loop, but it will be wordy. So the current impl is fine here.

Feel free to merge it into master.